src/orders/components/OrderDetailsPage/OrderDetailsPage.tsx

```diff
@@ -211,9 +228,30 @@ const OrderDetailsPage: React.FC<OrderDetailsPageProps> = props => {
               inline
               title={<Title order={order} />}
               cardMenu={
-                <CardMenu
-                  menuItems={[...selectCardMenuItems, ...extensionMenuItems]}
-                />
+                <>
+                  <Button
+                    data-test-id="graphiql-redirect"
+                    onClick={() => {
+                      const playgroundURL = new URL(process.env.API_URI);
+                      playgroundURL.hash = encodeGraphQLStatement({
+                        query: DefaultTestQuery,
+                        headers: "",
+                        operationName: "",
+                        variables: `{ "id": "${order.id}" }`,
+                      });
+                      window.open(playgroundURL, "_blank").focus();
+                    }}
```

Comment:
Why are we using `onClick` handler rather than providing `href`? Is there some performance concern?

FYI: You can change import of button to `components/Button` it uses `a` tag instead of `button` when `href` is passed (it still looks like a button)

Reply:
@witoszekdev The intention was to defer the URL compression (from `encodeGraphQLStatement`) to the time the button is clicked; also, eventually this default query could be dynamic. What's your take?

Reply:
My take was that links are better for accessibility: they behave differently than buttons (triggered on Enter rather than Space) + most a11y tools like screen readers can list every link on a page for faster navigation (e.g. VoiceOver on macOS can list all links on a page in rotor).
But on the contrary, if generating a link on every page visit would be too expensive performance wise then maybe using a button is a better idea - it still opens the GraphiQL in a new window, so the users should know what is going on. We could make it even more obvious to screen reader users by adding aria-label that explains what will happen if you click this button (e.g. "Opens example query in GraphiQL for this order in a new browser window")

Reply:
@witoszekdev the button has been converted into a link and moved under the details submenu, i.e. `CardMenu`